routes.yaml

```diff
@@ -0,0 +1,5 @@
+app_name: ch-processor-poc
```

Comment:
This needs to be changed to the following:
`
app_name: company-links-consumer
group: internalapi
weight: 900
routes:
   1: ^/company-links-consumer/healthcheck
`